### DISCUSSION TITLE
why don't make a open scratch btn without opening scratch

### DISCUSSION THREAD

**Discussion by Sreehari521** [ScratchAddons/ScratchAddons#3109] [Ideas]
## So we can automaticaly go to scratch without wasting time

**Comment by apple502j** [comment]
Can't you register bookmarks?

**Reply by Sreehari521** [reply]
Hmm....

**Reply by retronbv** [reply]
> Hmm....

Press control+d on the page you want to bookmark (chrome)

**Comment by Shluffy** [comment]
@Sreehari521 try @scratch-bookmarklets, or really go [here](https://github.com/Scratch-Bookmarklets/Scratch-Bookmarklets.github.io/blob/master/bookmarklets/goto-user/bookmarklet.js) or use this (`javascript: let user = prompt("Which user would you like to go to?"); window.location.href = "https://scratch.mit.edu/users/" + user + "/";`) to accomplish that
(I had to make some changes to not mess up the github code thing)
